src/main/java/com/wanderersoftherift/wotr/abilities/attachment/AbilitySlots.java

```diff
@@ -153,7 +154,8 @@ public int getSlotLimit(int slot) {
 
     @Override
     public boolean isItemValid(int slot, ItemStack stack) {
-        return stack.has(WotrDataComponentType.ABILITY);
+        return stack.is(WotrItems.ABILITY_HOLDER);
```

Comment:
It has been proposed that abilities would be crafted through essence infusion - meaning that empty ability holders will acquirable - and this change allows empty ability holders to be slotted. I would also be concerned that this will limit the ability to make variant ability sources (like the consumable ones).